### ISSUE TITLE
Recurring Events

### ISSUE THREAD

**Issue by taykcrane** [carlsednaoui/add-to-calendar-buttons#26]
Hey, any hope for support of recurring events via the RRULE property? (https://www.textmagic.com/free-tools/rrule-generator)

Would be super helpful! Thanks!

**Comment by drnikko** [comment]
Marking this as a duplicate of #18 and I'm working on it.  I've added RRULE to the ics link, but it's not working as expected.

**Comment by taykcrane** [comment]
@drnikki Thank you so much for working on this. Do you have a sense of when it'll be working, or no idea?

**Comment by drnikko** [comment]
Hi @taykcrane - if you're in a pinch you can look at the latest commit here: https://github.com/kanopi/add-to-calendar-buttons  this has a working ical/outlook recurrence addition (assuming that the text "RRULE" is already included in the string you're passing in, which it is for my use case).  

I'm currently blocked by trying to figure out the right google URL format for repeating events.  Once I can find the required parameters and formats, I'll be able to get that hooked up quickly. 

Once both are working, i'll open a PR here.

**Comment by taykcrane** [comment]
OK thanks so much for the update @drnikki. I'm most interested in the google use case, so I think I'll wait it out. If you remember, feel free to post in this thread or #18 when it's working.
